docs/releases/release-v28-0.md

```diff
@@ -0,0 +1,115 @@
+title: Release notes - V28.0 nano node
+description: Details of the V28.0 nano node release.
+
+# V28.0
+
+--8<-- "release-details-v28-0.md"
+
+---
+
+## Major updates
+
+### Bounded Block Backlog
+
+Designed by Piotr Wójcik, the Bounded Block Backlog (BBB) adds a maximum limit for unconfirmed transactions (currently 100,000), helping prevent resource exhaustion attacks & ensuring more consistent confirmation rates during high network load.
```

Comment:
>Designed by Piotr Wójcik
I don't think it's fair to say it's designed by me, the concept of BBB was there for a long time. I provided the first production ready implementation for it.